scss/modules/_colors.scss

```diff
@@ -21,13 +21,22 @@ $oc_gray_2: #D8D8D8;
 $oc_gray_3: #8B8A8A;
 $oc_gray_4: #5B5B5B;
 $oc_gray_5: #4E5A6B;
-$oc_gray_7: #0A0A0A;
-$oc_gray_8: #F8F8F8;
+$oc_gray_7: #637282;
+$oc_gray_8: #FBFAF7;
 // Reds
 $oc_red_dark: #9E001A;
 $oc_red_regular: #E04B53;
 $oc_red_light: #EC7A8D;
 // Charts
 $oc_chart_blue: #4EE4D6;
 $oc_chart_green: #4de148;
-$oc_chart_red: #D72A46;
\ No newline at end of file
+$oc_chart_red: #D72A46;
+
+$cr_text_blue: #4c67fc;
```

Comment:
not sure `cr` is quite as clear as `oc` was. Maybe just `carrot_...` is better here.